linkcheck/director/__init__.py

```diff
@@ -54,7 +52,7 @@ def check_urls (aggregate):
         raise
     except KeyboardInterrupt:
         interrupt(aggregate)
-    except thread_error:
+    except _thread.error:
```

Comment:
This nicely exposes that we're doing something that we shouldn't.  `_thread` is a private stdlib module in Python 3, so why are we using it directly, instead of using `threading`?

https://docs.python.org/3/library/_thread.html#_thread.error says
> Changed in version 3.3: This is now a synonym of the built-in RuntimeError.

so
```suggestion
    except RuntimeError:
```
and we can drop the import altogether.

Reply:
That's good. Done.